adventure/less/__init__.py

```diff
@@ -181,8 +203,6 @@ def game_over():
     check.stdout('You\'ve found the hidden path to victory. Unfortunately, '
                  'the passage is blocked by a barrier marked "Under Construction."',
                  regex=False)
-    check.stdout("\n")
-    check.exit(0)
```

Comment:
En dit? (kort hoor ;-))

Reply:
Deze code vereiste dat de game zichzelf afsloot als de speler won. Aangezien de FORCED 0 functionaliteit niet meer bestaat, sluit de game ook niet meer af als de speler wint. Vandaar dat die code weggehaald is, en deze check alleen nog checkt of de omschrijving van de ‘won’ room wordt geprint.

Reply:
Ah grappig dat het zo weinig verschil maakt :-)

Reply:
Is het hier nog nodig om te testen dat geen enkel "beweeg" commando meer werkt? Obviously kan dat niet gezien de data files. Maar moeten we dit dubbelchecken? @Jelleas @Qvdpost

Reply:
Zo nodig is dat redelijk eenvoudig toe te voegen:
```suggestion
    moves = ["EAST", "WEST", "NORTH", "SOUTH", "UP", "DOWN"]
    for move in moves:
        check.stdout("> ")
        check.stdin(move, prompt=False).stdout("Invalid command")
```